android-components/components/browser/menu/src/main/res/values/style.xml

```diff
@@ -68,6 +68,27 @@
         <item name="android:paddingEnd">@dimen/mozac_browser_menu_item_image_text_label_padding_start</item>
     </style>
 
+    <style name="Mozac.Browser.Menu.Item.Checkbox.Label" parent="Mozac.Browser.Menu.Item.Text">
```

Comment:
Should we not inherit from `Mozac.Browser.Menu.Item.ImageText.Label` here?  This is what you're replacing above and you could remove the reductant `android:layout_width` and `android:layout_height`. Note, `Mozac.Browser.Menu.Item.ImageText.Label` already inherits from from `Mozac.Browser.Menu.Item.Text`.